radar-commons/src/main/java/org/radarcns/producer/rest/BinaryRecordRequest.java

```diff
@@ -84,6 +84,9 @@ public void writeToSink(BufferedSink sink) throws IOException {
         binaryEncoder.writeInt(keyVersion);
         binaryEncoder.writeInt(valueVersion);
 
+        // do not send project ID and user ID; those are encoded in the serialization
+        binaryEncoder.writeIndex(0);
+        binaryEncoder.writeIndex(0);
```

Comment:
Why do we write twice?

Reply:
Once for the project ID, once for the user ID